index.html

```diff
@@ -176,11 +190,117 @@ <h2>Getting Started</h2>
         
         <div class="card">
             <h2>Compression</h2>
-            <ul>
-                <li></li>
-                <li>Tool 2</li>
-                <li>Tool 3</li>
-            </ul>
+            <div class="tool">
+                <h3>kkrunchy</h3>
+                <p>
+                    Written by Farbrausch, kkrunchy has been the de facto executable compression tool for 64k intros since its release in 2006.
+                </p>
+
+                <div class="pros-and-cons">
+                    <div class="pros">
+                        <h4>Pros</h4>
+                        <ul>
+                            <li>Efficient</li>
+                            <li>Fast</li>
+                            <li>Simple to use</li>
+                            <li>Source code available</li>
+                        </ul>
+                    </div>
+
+                    <div class="cons">
+                        <h4>Cons</h4>
+                        <ul class="cons">
+                            <li>Limited debugging information</li>
+                            <li>Generated binaries are generally flagged as malware by antivirus vendors</li>
+                            <li>Support for 32 bits only</li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="links">
+                    <h4>Links</h4>
+                    <ul class="tool-links">
+                        <li><a href="http://www.farbrausch.de/~fg/kkrunchy/">kkrunchy project page</a></li>
+                        <li><a href="https://www.pouet.net/prod.php?which=26088">kkrunchy on Pouët</a></li>
+                        <li><a href="https://demozoo.org/productions/38/">kkrunchy on Demozoo</a></li>
+                        <li><a href="https://github.com/farbrausch/fr_public/tree/master/kkrunchy">kkrunchy source code</a></li>
+                        <li><a href="https://fgiesen.wordpress.com/2011/01/24/x86-code-compression-in-kkrunchy/">Article: x86 code compression in kkrunchy</a></li>
```

Comment:
Do we really need all these links? The demozoo has little value, IMO. We can just send the visitor to the project page, which should have most of the information.